pkg/internal/containerd_conf.toml

```diff
@@ -64,8 +64,8 @@ version = 2
     unset_seccomp_profile = ""
 
     [plugins."io.containerd.grpc.v1.cri".cni]
-      bin_dir = "C:\\Program Files\\containerd\\cni\\bin"
-      conf_dir = "C:\\Program Files\\containerd\\cni\\conf"
+      bin_dir = "C:\\k\\cni"
+      conf_dir = "C:\\k\\cni\\conf"
```

Comment:
I think `conf` should be `config` in the `conf_dir` 
https://github.com/openshift/windows-machine-config-operator/blob/ce2d16caf8ba25c739b275bc6311617983cf44a0/pkg/windows/windows.go#L37-L40

Reply:
you are right, will fix this.

Reply:
Things like this is what is concerning when merging the PR without any e2e testing.